Source/NSURLProtocol.m

```diff
@@ -1519,8 +1523,8 @@ - (void) _got: (NSStream*)stream
 					  code: 0
 				      userInfo: nil];
 		  [self stopLoading];
-		  [this->client URLProtocol: self
-			   didFailWithError: e];
+		  [this->client URLProtocol: self didFailWithError: e];
+      DESTROY(this->client);
```

Comment:
Tabbing?

Reply:
True - but GNUstep is so messed up between people using spaces and tabs that I'm not going to waste my time on this.